collector.go

```diff
@@ -90,6 +91,21 @@ func (c metaCollector) Collect(ch chan<- prometheus.Metric) {
 	}()
 
 	config := c.config.ConfigForTarget(c.target, c.module)
+
+	if *isHashiCorp {
```

Comment:
instead of "isHashiCorp", it should be called "isVault". Hashcorp is hidden under the implementation of vault.